### PR TITLE
Modified strings for Querystring variables

### PR DESCRIPTION
**Pull request by Revathi-b**
Modified strings for Querystring variables @AnjanaValsan

### CODE REVIEW

File: MicrosoftSearch/manage-verticals.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Use the “|” operator to resolve multi-value variables. See the following tab
 
 #### Query String variables
 
-You can filter search results based on user interaction with SharePoint sites by using query string variables. Query string variables allows you to pass key-value pairs in the search URL to ensure the search results are customized based on those values. For example, suppose you have a SharePoint site that provides information on a project with a simple web part that shows in-progress tasks. Clicking on the "In-progress" web part, links users to the "Work items" search vertical, where the results are refined to show only items tagged as **InProgress**.
+You can use Query String variables to contextualize search results based on user interaction with SharePoint sites. Query string variables allows you to pass key-value pairs in the search URL to ensure the search results are customized based on those values. For example, suppose you have a SharePoint site that provides information on a project with a simple web part that shows in-progress tasks. Clicking on the "In-progress" web part, links users to the "Work items" search vertical, where the results are refined to show only items tagged as **InProgress**.
```
Comment (jeffkizn) [inline MicrosoftSearch/manage-verticals.md:156]: This is very abstract; the previous text was more human. Can you rephrase this to more common language?

**Review by jeffkizn — CHANGES_REQUESTED** [review]

File: MicrosoftSearch/manage-verticals.md (hunk 1 above)
Comment (Revathi-b) [reply MicrosoftSearch/manage-verticals.md:156]: @jeffkizn @AnjanaValsan 
Users were confused about this feature with the filters feature, and that’s the reason we thought of modifying the content.

Old content
You can use Query String variables to contextualize search results based on user interaction with SharePoint sites.

Below are the four proposed contents, and please feel free to share your thoughts on the content.

- You can modify search results based on user interaction with SharePoint sites using query string variables.
- Using query string variables, you can refine search results based on user interaction with SharePoint sites.
- Query String variables enable you to personalize search results based on how users interact with SharePoint sites.
- You can customize search results based on user interaction with SharePoint sites using query string variables.

File: MicrosoftSearch/manage-verticals.md [hunk 2]
```diff
@@ -182,7 +182,7 @@ Here are more examples of query string expansion.
 - Vertical modification and new verticals are not visible to guest users in an organization. 
 - Vertical re-ordering is not supported.
 - Vertical renaming for All tab is not supported in Microsoft Search in Bing.
-- Query string variables is not supported in Microsoft Search in Bing.
+- Query string variables is not supported in Microsoft Search in Bing and Microsoft Search in Office.
```
[the remark this answers is not in the document]
Comment (Revathi-b) [reply MicrosoftSearch/manage-verticals.md:185]: @jeffkizn Query string variables are only supported on SharePoint sites.
